manager/container.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 // Housekeeping interval.
+var enableLoadReader = flag.Bool("enable_load_reader", false, "Whether to enable cpu load reader")
```

Comment:
Apologies for overlooking that. The flag should ideally be deprecated. We
don't have a good policy around configuration as of now.

On Mon, Apr 4, 2016 at 2:54 PM, Thomas Orozco <notifications@github.com>
wrote:

> In manager/container.go
> <https://github.com/google/cadvisor/pull/1188#discussion_r58455971>:
>
> > @@ -42,6 +42,7 @@ import (
> >  )
> >
> >  // Housekeeping interval.
> > +var enableLoadReader = flag.Bool("enable_load_reader", false, "Whether to enable cpu load reader")
>
> I actually just moved that flag over from manager.go (i.e. it already
> exists in cAdvisor).
>
> We could definitely use disable_metrics to disable CPU load monitoring,
> but that'd break CLI compatibility with past versions.
>
> Let me know what you think,
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1188/files/002d8aae35acb23a69ddca2c1eedc29c75c36e2f#r58455971>
>